docker_test.go

```diff
@@ -122,3 +122,34 @@ func TestExternalVolume(t *testing.T) {
 	assert.Equal(t, ev2.Name, "")
 	assert.Equal(t, ev2.Provider, "")
 }
+
+func TestDockerPersistentVolume(t *testing.T) {
+	docker := NewDockerApplication()
+
+	pVol := &PersistentVolume{Size: 2048}
```

Comment:
I think we should drop lines 129 through 137: We want to use the convenience API calls to test our code. Thus, line 139 is how this test should start.

The constraints setup and assertions (including `EmptyConstraints`) should move below line 139 too.